src/App/ExtensionContainerPyImp.cpp

```diff
@@ -198,8 +198,7 @@ PyObject* ExtensionContainerPy::hasExtension(PyObject *args) {
 PyObject* ExtensionContainerPy::addExtension(PyObject *args) {
 
     char *typeId;
-    PyObject* proxy;
-    if (!PyArg_ParseTuple(args, "sO", &typeId, &proxy))
+    if (!PyArg_ParseTuple(args, "s", &typeId))

```

Comment:
@ickby
This will cause a problem for existing Python code. If 3rd party code uses two arguments it will raise an exception. To keep backward compatibility I suggest two allow an optional second argument. Its default value is null and in case it''s not null afterwards I would print a warning that the API has changed and the second argument will be removed after a while.

Reply:
Giving a deprecation warning seems more friendly to ad-don developers. I'm going to adopt the code.